packages/api/src/external/patient-external.ts

```diff
@@ -1,11 +1,12 @@
-import { PatientExternalData } from "../domain/medical/patient";
-import { getLinkStatusCW } from "./commonwell/patient";
-import { LinkStatusAcrossHIEs } from "./patient-link";
+// import { PatientExternalData } from "../domain/medical/patient";
+// import { getLinkStatusCW } from "./commonwell/patient";
+// import { LinkStatusAcrossHIEs } from "./patient-link";
 
-export function getLinkStatusAcrossHIEs(
-  externalData: PatientExternalData | undefined
-): LinkStatusAcrossHIEs {
-  return {
-    COMMONWELL: getLinkStatusCW(externalData),
-  };
-}
+// export function getLinkStatusAcrossHIEs(
+//   externalData: PatientExternalData | undefined
+// ): LinkStatusAcrossHIEs {
+//   return {
+//     COMMONWELL: getLinkStatusCW(externalData),
+//     CAREQUALITY: getLinkStatusCQ(externalData),
+//   };
+// }
```

Comment:
This function isnt being used anymore should i remove? It throws error no that ive added CQ

Reply:
The question for this and the other comments is whether we want to keep this logic in the PR as comments as it wont run anything or just remove it for now (ill add it to some other branch)